hybridse/src/udf/default_udf_library.cc

```diff
@@ -2679,6 +2720,28 @@ void DefaultUdfLibrary::InitUdaf() {
         )")
         .args_in<int16_t, int32_t, int64_t, float, double>();
 
+    RegisterUdafTemplate<StdUdafDef>("std")
```

Comment:
should we should support population standard deviation too ?

Reply:
support both `std` and `std_samp`